### ISSUE TITLE
Убрать использование излишней типизации

### ISSUE THREAD

**Issue by evgen-kent** [evgen-kent/AZATI_Workshop#76]
Вот пример 
https://github.com/evgen-kent/AZATI_Workshop/blob/adf9321097f9ea1d04fc1669ab363d0f0772a0cc/src/pages/Home/index.tsx#L7C18-L7C29

Так делать не надо

**Comment by Miracle-229** [comment]
JSX.Element убрать у компонентов ?

**Comment by puptup** [comment]
> JSX.Element убрать у компонентов ?

да, он бессмысленный, ибо тайпскрипт сам может понять, что будет возвращать функция
